shell/imports/server/accounts/ldap.js

```diff
@@ -76,7 +76,7 @@ LDAP.prototype.ldapCheck = function (db, options) {
       client = ldapjs.createClient({
         url: fullUrl,
         tlsOptions: {
-          ca: [_this.options.ldapsCertificate],
+          ca: [db.getLdapCaCert() || false],
```

Comment:
I was going off what was originally here (which was originally written by the meteor-accounts-ldap project that we forked from). In it, `_this.options.ldapsCertificate` defaults to `false`.